styles/AdvancedEditor.css

```diff
@@ -406,88 +406,79 @@ Next levels are shifted 'up' by 1
   position: static;
 }
 
-.bibliography-comment > .group-comment-icon {
-  top: -5px;
-  left: 9px;
-}
 
-.block-comment-button.bibliography-comment {
-  display: grid;
-  position: absolute;
-  top: -10%;
-  left: 1%;
+.comment-marker.selected-comment > svg > path, .comment-marker:active > svg > path {
+  stroke: #f7b314;
+  fill: #ffe0b2;
 }
 
-.block-comment-button {
+.comment-marker {
+  display: inline-flex;
+  width: 0;
+  z-index: 100;
+  top: -20px;
+  left: 0;
   cursor: pointer;
   position: relative;
-  height: 0;
-  width: 0;
 }
 
-.block-comment-button:active > svg > path {
-  stroke: #f7b314;
-  fill: #ffe0b2;
-}
-
-.selected-comment > svg > path {
-  stroke: #f7b314;
-  fill: #ffe0b2;
+.block > div.comment-marker, .figure-block > div.comment-marker {
+  top: -25px;
+  left: 100%;
 }
 
-.block-comment-button.block-comment {
-  z-index: 10;
-  left: 93%;
+.figure-block div.comment-marker {
+  grid-column: 3;
 }
 
-.block-comment > .comment-icon,
-.group-comment-icon {
-  position: absolute;
-  top: 16px;
-}
-
-.block-comment > .group-comment-icon {
-  left: 7px;
-  top: 10px;
+figure.block > .comment-marker {
+  float: right;
+  top: 0;
+  left: 0;
 }
 
-.block-comment-button.figure-comment {
-  position: absolute;
-  right: 3%;
-  z-index: 10;
+section > .comment-marker {
+  top: 12px;
+  float: right;
+  left: -64px;
 }
 
-.figure-comment > .comment-icon {
-  position: absolute;
-  top: 4px;
+span.comment-marker {
+  top: -38px;
+  left: -10px;
 }
 
-.figure-comment > .group-comment-icon {
-  top: 0;
-  left: 10px;
+.citation-wrapper + .comment-marker {
+  left: -2px;
 }
 
-.block-comment-button.inline-comment {
-  z-index: 10;
-  left: 100%;
+.keywords-group > .comment-marker {
```

Comment:
that needs to be `.keyword-group > .comment-marker`

Reply:
Fixed.